DigitalOffice/Tests/UserServiceTests/Commands/GetUserByEmailCommandTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.UserService.Commands;
+using LT.DigitalOffice.UserService.Commands.Interfaces;
+using LT.DigitalOffice.UserService.Database.Entities;
+using LT.DigitalOffice.UserService.Mappers.Interfaces;
+using LT.DigitalOffice.UserService.Models;
+using LT.DigitalOffice.UserService.Repositories.Interfaces;
+using LT.DigitalOffice.UserService.Validators;
+using Moq;
+using NUnit.Framework;
+
+namespace LT.DigitalOffice.UserServiceUnitTests.Commands
+{
+    public class GetUserByEmailCommandTests
```

Comment:
I think these tests are not testing the command.

Reply:
Why not?

Reply:
Check if repository or mapper will return exception.

Reply:
Fixed